CMakeLists.txt

```diff
@@ -123,7 +123,12 @@ add_library(common STATIC ${common_srcs})
 # OpenGL
 find_package(OpenGL REQUIRED)
 find_package(GLUT REQUIRED)
-include_directories(${OPENGL_INCLUDE_DIRS} ${GLUT_INCLUDE_DIRS})
+find_package(GLM REQUIRED)
```

Comment:
I'm working on an Arch Linux box, and I got the same error before I installed GLM. After installing GLM with the package manager though, the CMake error went away.

I think GLM comes with a .cmake, rather than CMake itself. Do you have GLM installed on your system yet?